### PR TITLE
Tanzania (National Assembly): refresh data

### PR DESCRIPTION
**Pull request by everypoliticianbot**
Automated data refresh for Tanzania - National Assembly

#### Output

```
Add memberships from sources/manual/term4.csv
Add memberships from sources/morph/term5.csv
Merging with sources/morph/wikidata.csv
Data Mismatches
  ☁ Mismatch in birth_date for 9a143f85-3c51-4011-82dd-d29578ebdfd1 (1961-01-01) vs 1961-07-01 (for Q16194369)
  ☁ Mismatch in birth_date for 12b02f92-44fd-4d94-aa85-4544a390d19f (1963-05-23) vs 1963-05-25 (for Q16195184)
  ☁ Mismatch in birth_date for 4438b413-f970-4a3c-b850-6da88f774750 (1970-10-10) vs 1971-10-10 (for Q16213580)
  ☁ Mismatch in birth_date for 0b9d2462-9e5c-49be-9432-35de19f49cb7 (1986-07-08) vs 1985 (for Q16733187)
  ☁ Mismatch in birth_date for baca35ba-3907-47ab-8a22-c5a7418e2293 (1948-12-24) vs 1947-12-24 (for Q4756599)
  ☁ Mismatch in birth_date for 72277650-ab84-4e53-9292-60f99ee818ad (1959-07-06) vs 1956-08-09 (for Q6399650)
  ☁ Mismatch in birth_date for b2cd4f01-c3f8-4f2a-84e2-5ee84e63c8e7 (1975-08-05) vs 1975-05-08 (for Q6893059)
  ☁ Mismatch in birth_date for d45e9019-e89c-41de-95c8-a2cd2e123878 (1960-01-22) vs 1960-01-21 (for Q9032315)
* 4 of 245 unmatched
	{:id=>"Q461710", :name=>"Asha-Rose Migiro"}
	{:id=>"Q7369953", :name=>"Rostam Aziz"}
	{:id=>"Q16145336", :name=>"Juma Ali"}
	{:id=>"Q13123140", :name=>"Jeremiah Solomon Sumari"}
Adding GenderBalance results from sources/gender-balance/results.csv
  ⚥ data for 1048; 604 added

GenderBalance Mismatches
    ☁ Mismatch for 6aab5137-9b15-4f5a-9e23-363439ea1fcb Lolesia Jeremiah Maselle Bukwimba (Was: male | GB: female)
    ☁ Mismatch for 161e5986-f0a9-409c-9a52-db6263d73383 Waride Bakari Jabu (Was: male | GB: female)
    ☁ Mismatch for 65770683-8ad7-47c1-9b53-3e95982bba60 Faith Mohamed Mitambo (Was: male | GB: female)
    ☁ Mismatch for 6aab5137-9b15-4f5a-9e23-363439ea1fcb Lolesia Jeremia Maselle Bukwimba (Was: male | GB: female)

Top identifiers:
  241 x wikidata
  10 x freebase
  5 x viaf
  3 x sudoc
  2 x isni

Creating names.csv
Persons matched to Wikidata: 241 ✓ | 564 ✘
  No wikidata: Sadiq Ahmad Suleiman (5653fcda-f1c3-4f0d-8c6c-8d1738fadf53)
  No wikidata: Margareth Agnes Mkanga (8262e3b7-fb2a-4d12-89b3-1e2237771ed6)
  No wikidata: Shareef Abdulkadir (076437f1-b1f0-48db-9fac-f65e4c57700d)
  No wikidata: Cheyo M. Isaac (7989c344-bff4-46a7-8aab-32de89be3bc4)
  No wikidata: John Wegesa Heche (2b756963-1eb3-4194-b5d7-92d9b65e4b14)
  No wikidata: Margareth Simwanza Sitta (dad1747b-8b63-4bfb-8706-c4bfd9423425)
  No wikidata: Azza Hillal Hamad (a80d976d-018c-4650-bb88-c0b2049d04f9)
  No wikidata: Pindi Hazara Chana (ffad6049-b0d7-4f97-8f11-a0f7633e0e04)
  No wikidata: Munde Abdallah Tambwe (41bac91f-2a77-4cfa-9a78-03d792448130)
  No wikidata: Dr. Susan Alphonce Kolimba (2797adda-ca66-4350-a652-ef139098a4a0)
Parties matched to Wikidata: 10 ✓ | 2 ✘
  No wikidata: - (party/-)
  No wikidata: Civil United Front (party/civil_united_front)

```

### CODE REVIEW

**Comment by ondenman** [comment]
Looks like the problem is with the scraper failing on morph. I ran the scraper locally for this PR but had to do so twice as the first attempt failed with a timeout.

**Comment by tmtmtmtm** [comment]
@ondenman this is a little hard to follow because of the double commits. In a case like this I think you'd have been better dropping your first lot and then redoing the second time with a full clean version of the data.

But from what I can tell, a fresh build with 'correct' data from this scraper (though not the version currently on morph), is clean, with no changes. So no value in merging this. 👍
